src/js/components/charts/DialChart.js

```diff
@@ -80,14 +78,20 @@ var DialChart = React.createClass({
     }
 
     // Zero-length defaults are populated with actual data if available
-    var namedSlices = _.indexBy(slices, 'name');
-    var namedData = _.indexBy(data, 'name');
-    var normalizedNamedData = _.extend(namedSlices, namedData);
-    return _.values(normalizedNamedData);
+    var namedSlices = slices.reduce(function (indexByName, slice) {
+      indexByName[slice.name] = slice;
+      return indexByName;
+    }, {});
+    var namedData = data.reduce(function (indexByName, datum) {
+      indexByName[datum.name] = datum;
+      return indexByName;
+    }, {});
+    var normalizedNamedData = Object.assign({}, namedSlices, namedData);
+    return Object.values(normalizedNamedData);
```

Comment:
I find this way more confusing to read ![](http://cl.ly/0u0x1O3v1Q0r/Image%202016-05-03%20at%203.17.26%20PM.png)

this isn't much better :'D ![](http://cl.ly/3R463b3l0J2H/Image%202016-05-03%20at%203.18.06%20PM.png)

I will concede to this though: ![](http://cl.ly/341x1i0e2p1R/Image%202016-05-03%20at%203.18.40%20PM.png)